Tests/kaas/k8s-version-recency/k8s-version-recency-check.py

```diff
@@ -83,13 +83,13 @@ def print_usage():
     print("""
 K8s Version Recency Compliance Check
 
-Usage: k8s-version-recency-check.py [-h] [-k|--kubeconfig PATH/TO/KUBECONFIG]
+Usage: k8s-version-recency-check.py [-h] [-c|--config PATH/TO/CONFIG] -k|--kubeconfig PATH/TO/KUBECONFIG
 
 The K8s version recency check returns 0 if the version of the tested cluster is still acceptable, otherwise
 it returns 1 for an out-of-date minor version, 2 for an out-of-date patch level version or 3 if the currently
 used version should be updated due to a highly critical CVE.
```

Comment:
For me, it is a little bit different as you write in header:
```
Return codes:
0: Version is inside the recency window
1: Error during script execution
2: Version isn't inside the recency windows anymore
3: Version used contains a critical CVE
```

Reply:
Yeah you're right, that wasn't updated quit some time.